src/anshuKJun20/FibonacciSeries.java

```diff
@@ -0,0 +1,26 @@
+package anshuKJun20;
+
+public class FibonacciSeries {
+	
+	private void findFibbonacciSeries(int count) {
+		System.out.println('\n'+"The fibonacci Series upto "+count+":");
+	    int number1 =0 ,number2 = 1;
+		display(number2);
+	    for(int index=0;index<count;index++) {
+	       int number3 = number1 +   number2;
+			display(number3);
+			number1 = number2;
+			number2 = number3;
+		}
+	}
+	
+	void display(int number3) {
+			System.out.print(number3+" ");
```

Comment:
If n=5,means 5 numbers should be present in series.likewise if n=10 then 10 numbers should be displayed in series.
Means:-
Fibonacci series for n=5 should display
1,1,2,3,5
Fibonacci series for n=10  should  display
1,1,2,3,5,8,13,21,34,55

Change logic accordingly

Reply:
ya i have done this according to n= 5.but have doubt because, @maulik  output was 1 1 2 3 5 8. so that,s why i did upto loop (i=0 ; i<5;i++) @maulik , please correct us. what means n = 5 . in this series.